src/assets/styles/sass/styles.scss

```diff
@@ -0,0 +1,66 @@
+.projects-listing {
+display: grid;
+grid-template: repeat(2, 1fr) / repeat(3, 1fr);
+place-items: center;
+column-gap: 1rem;
+row-gap: 1rem;
+}
+
+.project-card {
+    height: 100%;
+    width: 100%;
+    border: 1px solid rgb(213, 213, 213);
+    position: relative;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/descholar-ceo/portfolio/pullRequest?prid=5731031)